### ISSUE TITLE
Display timezone next to dates in published articles

### ISSUE THREAD

**Issue by nilsml** [equinor/energyvision#888]
Q: Is this change only for news and events templates, or should we add it to the event cards as weil?

## Acceptance criteria
- Published content in one time zone is shown with correct time zone for the place in which it is viewed

**Comment by fernandolucchesi** [comment]
![image](https://user-images.githubusercontent.com/23337266/172832319-ebfe1c4e-e656-4eb0-aff9-dde21e639bb2.png)

**Comment by fernandolucchesi** [comment]
@bhavi25, looks good! Only missing thing is the use of feature flags. Can you please add it to this?

**Comment by fernandolucchesi** [comment]
@meols @jorgol can any of you review this?

**Comment by meols** [comment]
@fernandolucchesi - Looks good :smile: Testet on test (where it doesn't show) and dev (where it works). Great if you add a link to where to test :wink:

**Comment by fernandolucchesi** [comment]
> @fernandolucchesi - Looks good 😄 Testet on test (where it doesn't show) and dev (where it works). Great if you add a link to where to test 😉

Nice! It was @bhavi25 first issue. 🎊 

We also added a new column to the board (Feature flag / deploy). Now issues need to be moved there before closing. :)
